AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_CubeMapCaptureAdded.py

```diff
@@ -87,22 +108,39 @@ def AtomEditorComponents_CubeMapCapture_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.removal_undo, cubemap_capture_entity.has_component(AtomComponentProperties.cube_map_capture()))
 
-        # 5. Delete Cubemap_Capture entity.
+        # 5. Set Specular IBL property: Very High
+        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL'), value=SPECULAR_IBL_QUALITY['Very High'])
+        Report.result(Tests.specular_ibl_property_set_to_very_high, cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL')) == SPECULAR_IBL_QUALITY['Very High'])
+
+        # 6. Set Specular IBL property: Very Low
+        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL'), value=SPECULAR_IBL_QUALITY['Very Low'])
```

Comment:
for dictionary enum values we frequently use a `for quality in SPECULAR_IBL_QUALITY:`
then you can set/get with value=SPECULAR_IBL_QUALITY[quality] and iterate over all the possible options
if you do that you would have to set your tuple locally with f-string local_tuple = (f"Specular IBL quality set to {quality}", f"failed to set Specular quality to {quality}")
doing a for loop is optional but allows us to test all the values in addition to just the 'Very High' and 'Very Low' boundaries.

Reply:
Added in the loop and removed the "Very High test"